frontend/src/components/Reaction.js

```diff
@@ -47,9 +44,10 @@ const Reaction = props => {
   };
 
   return (
-    <p onClick={toggleReaction}>
-      {type}: {reactionNumber}
-    </p>
+    <div className="reaction" onClick={toggleReaction}>
+      <Icon width="28px" fill={hasReacted ? "green" : "#8a8a8a"} />
```

Comment:
move logic from `fill` to const , and use hex for green color
suggestion => `const iconFillColor = hasReacted ? "green" : "#8a8a8a"`
```suggestion
      <Icon width="28px" fill={iconFillColor} />
```